### DISCUSSION TITLE
[needs verification] User not receiving notification when his comments are replied to.

### DISCUSSION THREAD

**Discussion by vegeta123254** [posativ/isso#899] [Q&A]
Whenever a new comment is made by the **user** , Example Foo is notified by email.

However when there is a reply to the comment, the **commentor(user)**  is not being notified, even though`reply-notifications= true` in the server and the client being configured.
I see the checkbox though.

Server side:
```
[general]
reply-notifications= true

[smtp]
host = smtphost 
port = 25
security = none
from = "Example Foo" <examplefoo@tld.com> 
to = user@tld.com
timeout = 10
```

Client side:

```
        data-isso-reply-notifications="true"
```
Am I missing something?

**Comment by thomasjsn** [comment]
I just set this up and it works. Could your SMTP server be limiting which addresses you can send to? That happened with my AWS SES implementation.

**Comment by ix5** [comment]
@vegeta123254 could you kindly verify whether the issue is indeed caused by what @thomasjsn suggested?
Without further logs/setup info it will be hard to verify and fix your issue.
